Sources/StreamChatUI/ChatChannelList/ChatChannelUnreadCountView.swift

```diff
@@ -48,7 +48,7 @@ open class _ChatChannelUnreadCountView<ExtraData: ExtraDataTypes>: _View, UIConf
     }
     
     override open func updateContent() {
-        isHidden = content.mentionedMessages == 0 && content.messages == 0
+        alpha = content.mentionedMessages == 0 && content.messages == 0 ? 0 : 1
```

Comment:
Why this change?

Reply:
The unreadCountView when is empty, we just want to hide the opacity. But because this view is present in a container stack view, that will hide the view from the container, and re-layout the views inside, make the timestamp label to the centre of the container, which we don't want.